lib/modules/hosts/wikipedia.js

```diff
@@ -7,16 +7,31 @@ export default {
 	attribution: false,
 	detect: ({ href }) => (/^https?:\/\/([A-z]{2}|simple)?\.(?:m\.)?wikipedia\.org\/wiki\/(.+)/i).exec(href),
 	async handleLink(href, [, country, article]) {
```

Comment:
BTW, this isn't a country code but a language code, but it doesn't matter much. :-)